yt/data_objects/tests/test_profiles.py

```diff
@@ -706,6 +706,15 @@ def test_export_astropy():
     assert "velocity_x" not in at2.colnames
     assert_equal(prof.x.d[prof.used], at2["radius"].value)
     assert_equal(prof[("gas", "density")].d[prof.used], at2["density"].value)
+    at3 = prof.to_astropy_table(fields=("gas", "density"), include_std=True)
+    assert_equal(prof[("gas", "density")].d, at3["density"].value)
+    assert_equal(
+        prof.standard_deviation[("gas", "density")].d, at3["density_stddev"].value
+    )
+    assert (
+        prof.standard_deviation[("gas", "density")].units
+        == YTArray.from_astropy(at3["density_stddev"]).units
```

Comment:
shouldn't the field type also be included in the key (`"gas_density_stddev"` instead of `"density_stddev"`) ?

Reply:
@neutrinoceros so the field type is not included already even when we do not include the standard deviation (I don't remember why). So you would access the mean of the profile using `at3["density"]`

Reply:
ok, at least it's consistent.

Reply:
We could add the field type, but I'm a bit leery of using `_` to connect them since `_` could appear in field type strings (and does appear in field name strings), making parsing a bit difficult in some cases. 

Is there another sensible character to use to indicate the breaking of the field type and name? Maybe even `,` is ok? I don't know (cc @matthewturk)

Reply:
I guess that would be a breaking change, so I'm actually -1 on this. Let's keep it as is for now.